tests/nlu/extractors/test_crf_entity_extractor.py

```diff
@@ -41,7 +42,16 @@ def test_crf_extractor(spacy_nlp, ner_crf_pos_feature_config):
         ),
     ]
 
-    extractor = CRFEntityExtractor(component_config=ner_crf_pos_feature_config)
+    extractor = CRFEntityExtractor(
+        component_config={
+            "features": [
+                ["low", "title", "upper", "pos", "pos2"],
+                ["low", "suffix3", "suffix2", "upper", "title", "digit", "pos", "pos2"],
+                ["low", "title", "upper", "pos", "pos2"],
+            ],
+            RANDOM_SEED: 1,
```

Comment:
Do we need to add smaller number of epochs here? i remember this worked with 75 epochs the last time I tested with a particular speed. Can leave it out if the speed difference is marginal.